src/Department.java

```diff
@@ -1,7 +1,55 @@
 package application;
 
+import java.util.Random;
+
 public class Department {
+	public String name="Test";
+	public int id=123;
+	public String country="Neverland";
+	public String city="Gotham";
```

Comment:
If you use type encapsulation make the variables `private` otherwise the getter and setter do not make sense.